### PR TITLE
View/feature

### PR DESCRIPTION
**Pull request by denscholar**
## What was implemented in this PR

- I used methods that I created in the [Blog App - processing data in models] to create my views
- Created some dummy data and seed to it so I can be able to view the data

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -1,5 +1,11 @@
...
+  def index
+    @user = User.find(params[:user_id])
+    @post = Post.where(author_id: @user.id).order(created_at: :desc)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:id])
+    @comment = Comment.where(post_id: @post.id).order(created_at: :desc)
+  end
```
Comment (gemmen29) [inline app/controllers/posts_controller.rb]: - [ ] Kindly fix the error in show post details
image

File: db/seeds.rb [hunk 2]
```diff
@@ -6,7 +6,29 @@
...
+first_user = User.create(name: 'Tom', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Teacher from Mexico.')
+second_user = User.create(name: 'John Smith', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Doctor from Spain.')
+third_user = User.create(name: 'Noah', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Doctor from Mexico.')
+fourth_user = User.create(name: 'Ann Charlotte', photo: 'https://images.unsplash.com/photo-1508921912186-1d1a45ebb3c1', bio: 'Teacher from Mexico.')
+
+# Posts
+first_post = Post.create(author_id: first_user.id, title: 'Hello', text: 'This is my first post')
+second_post = Post.create(author_id: first_user.id, title: 'Hi', text: 'This is my second post')
+third_post = Post.create(author_id: first_user.id, title: 'Hi there', text: 'This is my third post')
+fourth_post = Post.create(author_id: first_user.id, title: 'Hello!', text: 'This is my fourth post')
```
Comment (gemmen29) [inline db/seeds.rb]: - [ ] [OPTIONAL] Please add the `commentsCounter` and `likesCounter` for each post in seeds so it's inserted in database.
```
first_post = Post.create(author_id: first_user.id, title: 'Hello', text: 'This is my first post', commentsCounter: 0, likesCounter: 0)
```

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @denscholar, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 💯 
- No linters errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: db/migrate/20220714204023_create_users.rb [hunk 3]
```diff
@@ -4,7 +4,7 @@ def change
       t.string :name
       t.string :photo
       t.text :bio
-      t.integer :postCounter
+      t.integer :postsCounter
```
Comment (elisha2kyakpo1) [inline db/migrate/20220714204023_create_users.rb:7]: - [ ] [OPTIONAL] Ruby naming convention should be followed every time, the attribute `postsCounter` should read `post_counter`.

**Review by elisha2kyakpo1 — APPROVED** [review]
Hi 👋 @denscholar, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- You did well to make the changes requested by the previous reviewer 👍

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**